build/dependencies.props

```diff
@@ -1,6 +1,9 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <AspNetCore01Version>0.1.0-*</AspNetCore01Version>
```

Comment:
Rather annoying that 03 is used for Diagnostics.Elm, Proxy and Buffering but sure I'll name them.